### PR TITLE
PP9-11809 - Angular SDK: Share creation > support language selection

### PR DESCRIPTION
**Pull request by tc4rdpp**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html [hunk 1]
```diff
@@ -72,6 +72,18 @@ <h3 class="header">
           <pp-date-picker [parentForm]="sharedContentForm" [placeHolderMessage]="'DD/MM/YYYY'"></pp-date-picker>
         </li>
       </ul>
+      <ul>
+        <li class="label-content">
+          <label for="language">{{ 'ShareContentDialog.Language' | pptranslate }}:</label>
+        </li>
+        <li class="input-content language">
+          <mat-select [formControl]="languageFormControl">
+            <mat-option *ngFor="let language of languageService.languages" [value]="language.ietf">
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html:81]: use share languages

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html [hunk 2]
```diff
@@ -72,6 +72,18 @@ <h3 class="header">
           <pp-date-picker [parentForm]="sharedContentForm" [placeHolderMessage]="'DD/MM/YYYY'"></pp-date-picker>
         </li>
       </ul>
+      <ul>
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html:75]: Hide if only one language available

Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html:75]: Align date field

Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html:75]: Move items scroll bar to the tab content

Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html:75]: Remove long subject validation

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html (hunk 1 above)
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html (hunk 2 above)
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html]: Done

Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html]: Done

Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html]: Done

Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html]: Done

**Review by neotek — APPROVED** [review]
LGTM

**Pull request merged**